content-src/asrouter/components/ConditionalWrapper/ConditionalWrapper.jsx

```diff
@@ -3,5 +3,7 @@
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 // lifted from https://gist.github.com/kitze/23d82bb9eb0baabfd03a6a720b1d637f
-export const ConditionalWrapper = ({ condition, wrap, children }) =>
+const ConditionalWrapper = ({ condition, wrap, children }) =>
```

Comment:
Why was this change needed?

Reply:
it wasn't technically needed, but since it's a single-serve component it seemed simpler to move from a named export to a default. happy to revert if this will cause confusion.

Reply:
No need makes sense to me :+1: